docs/faq.md

```diff
@@ -38,3 +38,43 @@ Project Mu makes liberal use of multiple repositories due to the mixture of requ
 repos are split for technical reasons, some for organizational, and some for legal.
 
 For details, see "Repo Philosophy" in [What and Why](WhatAndWhy/overview.md#repo-philosophy).
+
+## Is there any stronger guidance than "Less is More"?
+
+Since Mu is derived from Edk2, the Edk2 coding standards are a foundation, as can the existing code in the 

```

Comment:
`as are the existing code` instead of `as can the existing code`?

Reply:
Fixed.